playground/mocks/api/v1/idx/challenge/data/factor-required-email-with-options.json

```diff
@@ -0,0 +1,102 @@
+{
+  "stateHandle": "01pCiYqxWzm-iaFdL2W26KHFB-myLMJk2JBrLWmPxF",
+  "version": "1.0.0",
+  "expiresAt": "2019-08-14T20:42:04.000Z",
+  "step": "AUTHENTICATE",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "challenge-factor",
+        "href": "http://localhost:3000/api/v1/idx/challenge",
```

Comment:
curious shall this be `/api/v1/idx` or `/api/v1/idx/challenge`?

Reply:
Need to confirm with API folks

Reply:
According to the postman collection its `idx/challenge` 

<img width="1029" alt="Screen Shot 2019-08-20 at 11 48 55 AM" src="https://user-images.githubusercontent.com/23267876/63375458-a8deae80-c340-11e9-9c00-389e0b40b681.png">